nvm.sh

```diff
@@ -2349,17 +2368,17 @@ nvm() {
   COMMAND="${1-}"
   shift
 
-  # initialize local variables
+  # Initialize local variables
   local VERSION
   local ADDITIONAL_PARAMETERS
 
   case $COMMAND in
     'help' | '--help')
       local NVM_IOJS_PREFIX
       NVM_IOJS_PREFIX="$(nvm_iojs_prefix)"
-      local NVM_NODE_PREFIX
-      NVM_NODE_PREFIX="$(nvm_node_prefix)"
+      local NVM_NODE_PREFIX="$(nvm_node_prefix)"
```

Comment:
this must be reverted; ksh doesn't support `local` declarations, so they always have to be on their own line.

Reply:
I reverted the change in a new commit.